pdk/1.0/modules/ROOT/pages/policies-pdk-create-project.adoc

```diff
@@ -0,0 +1,140 @@
+= Creating a New Project
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+To begin developing a new custom policy with the Policy Development Kit (PDK), generate a new custom policy project.
+
+To create a new project, complete the following tasks:
+
+. <<before-you-begin>>
+. <<create-a-new-policy-project>>
+. <<setup-the-pdk-build-environment>>
+. Review <<project-structure>>
+
+[[before-you-begin]]
+== Before You Begin
+
+Before you can create a custom policy project, ensure that you have installed all xref:policies-pdk-prerequisites.adoc[PDK prerequisites].
+
+[[create-a-new-policy-project]]
+== Create a New Policy Project
+
+To create a custom policy project, run the PDK CLI `create` command, replacing `<my-custom-policy>` with the name of your new policy:
```

Comment:
Is this supposed to be a numbered step? Are all the action items in this section supposed to be steps?